x-pack/plugin/core/template-resources/src/main/resources/ecs@mappings.json

```diff
@@ -23,6 +23,26 @@
             "unmatch_mapping_type": "object"
           }
         },
+        {
+          "ecs_event_original_non_indexed_keyword": {
+            "mapping": {
+              "type": "keyword",
+              "index": false,
+              "doc_values": false
+            },
+            "path_match": "event.original"
+          }
+        },
+        {
+          "ecs_x509_public_key_exponent_non_indexed_keyword": {
+            "mapping": {
+              "type": "keyword",
```

Comment:
it seems this is a long according to ECS? https://www.elastic.co/guide/en/ecs/8.11/ecs-x509.html#field-x509-public-key-exponent I really wonder if we should cover this "edge" case.

Reply:
I think I agree. ECS doesn't mention whether it's indexed or not - so I guess indexing is fair game (and it's left to the user to overwrite this)?